.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -3,15 +3,27 @@ name: 🎉 Feature request
 about: Suggest an idea to improve GitHub Desktop
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I have an issue when [...]
+## Goal
+A clear and concise description of what the problem and request are. Ex. I have an issue when [...problem...]. I'd like [...feature...] so that [...reason...]
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen. Add any considered drawbacks.
+#### Background
+Any related issues or any preceding features that this would depend on. (linked from desktop repository)
 
-**Describe alternatives you've considered**
+## Solution
+A clear and concise description of what you want to happen, implementation-wise. Add any considered drawbacks. If you can, explain how users will be able to use this.
+
+#### Alternatives Considered
 A clear and concise description of any alternative solutions or features you've considered.
 
-**Teachability, Documentation, Adoption, Migration Strategy**
-If you can, explain how users will be able to use this. This will help the core team understand how the feture would work, and assist with future work like documentation.
-Screenshots or mockups here are a great help!
\ No newline at end of file
+## Teachability, Documentation, Adoption, Migration Strategy*
+This will help the core team understand how the feature would work, and assist with future work like documentation.
+Screenshots or mockups here are a great help! Here are some example subheaders:
+
+#### Task List

```

Comment:
This feels like an implementation-specific thing - should we put this up-front or only add this after we've been able to come to an agreement on things?

Reply:
👍 you're right, no point in listing it unless the solution is 👍ed. added `(once solution is agreed upon)` after this line

Reply:
If this template is the same one that everyone fills in, does this add value?